spec/errors_spec.js

```diff
@@ -0,0 +1,20 @@
+
+var modal = require('./helpers/modal');
+var prefs = require('./helpers/user_prefs');
+var authoring = require('./helpers/authoring');
+var monitoring = require('./helpers/monitoring');
+
+describe('errors', () => {
+    it('SDESK-3965: render modals with translated specs', () => {
```

Comment:
Write a clearer name for the test, for example `it('displays default language strings in the modals when no translations are available')`